app/controllers/profiles_controller.rb

```diff
@@ -4,33 +4,8 @@ class ProfilesController < ApplicationController
   def show
   end
 
-  def edit
-  end
-
-  def update
-    user = User.find(current_user.id)
-    if !existing_email?
-      if user.update!(user_params)
-        flash[:notice] = "Your information has successfully been updated."
-        redirect_to "/profile"
-      end
-    else
-      flash[:error] = "That email is already in use."
-      render :edit
-    end
-  end
-
   private
 
-  def existing_email?
-    User.exists?(email: params[:email]) && params[:email] != current_user.email
-  end
-
-  def user_params
-    params[:password_digest] = current_user.password_digest
-    params.permit(:name, :address, :city, :state, :zip, :email, :password_digest)
-  end
-
```

Comment:
Good idea of moving these over to user_controller in order to make our application more readable.

Reply:
Agreed, makes more sense